notebooks/entrypoint.sh

```
@@ -17,9 +17,13 @@ settings = {'headers': {'Content-Security-Policy': csp}}
 print(json.dumps(settings))
 ")
 
+# Enable the custom Voila extension for service_token support
+export PYTHONPATH=/app/notebooks:$PYTHONPATH
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)